src/shared_utilities/drivers.jl

```diff
@@ -230,17 +233,21 @@ function surface_fluxes_at_a_point(
         sc;
         tol_neutral = SFP.cp_d(surface_flux_params) / 100000,
     )
-    # Land needs a volume flux of water, not mass flux
-    vapor_flux =
-        SurfaceFluxes.evaporation(surface_flux_params, sc, conditions.Ch) /
-        _ρ_liq
-    return (
-        lhf = conditions.lhf,
-        shf = conditions.shf,
-        Ch = conditions.Ch,
-        vapor_flux = vapor_flux,
-        r_ae = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc)),
-    )
+    grav::FT = LSMP.grav(earth_param_set)
```

Comment:
This code accounts for the surface resistance in SHF, LHF, and E. Previously, this correction was made outside of this function.